README.asciidoc

```diff
@@ -539,8 +539,10 @@ Changes
 
  * `u`: undo last change
  * `<c-k>`: move backward in history
+ * `<a-u>`: undo selection changes. Introduced in v2023.07.29
```

Comment:
Those "Introduced in ..." are unnecessary, can you remove them please.

Reply:
No problem, done 👍.